lib/web/server.cpp

```diff
@@ -338,17 +354,17 @@ string Socialite::Web::Server::serverAPI(HttpHeader* header, HttpResponseHeader*
 
 // the meat of the actual processing
 void Socialite::Web::Server::handleConnection(NetStream* stream) {
-	int error = 0;
 	HttpHeader header;
-	HttpResponseHeader rHeader;
-	struct stat fileStat;
-	HostConfig* currentHost;
-	smatch matcher;
 	int connectionUse = 0;
-
 	stream->setTimeout(5 * 60); // die after 5 minutes of inactivity
 
 	do { // process each request as it comes in if KEEP_ALIVE, otherwise fall through
+		int error = 0;
+		HttpResponseHeader rHeader;
```

Comment:
Reset on each new connection within the same `NetStream` connection. Otherwise some bits of the previous request were carrying over to the following ones on the same connection.